docs/pages/blog/turbo-1-2-0.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Turborepo 1.2
+date: 2022/04/06
+description: Turborepo 1.2 introduces improved task filtering, artifact signing and integrity, and more!
+tag: web development
+ogImage: /images/blog/turbo-1-2-0/twitter-card.png
+---
+
+# Turborepo 1.2
+
+import { Authors } from '../../components/Authors'
+
+<Authors authors={['jaredpalmer', 'becca__z', 'gaspargarcia_', 'gsoltis']} />
+
+Since releasing Turborepo v1.1 in late January, we've seen incredible adoption and community growth:
+
+- 6.5k+ GitHub Stars
+- 140k+ weekly NPM downloads (up 2x since our [last blog post for v1.1](/blog/turbo-1-1-0))
+- 90+ OSS contributors
+- 900+ members of the [Turborepo Community Discord](https://turborepo.org/discord)
+- 1.6 years of Time Saved through Remote Caching on Vercel, saving +2.5 months per week now
+
+We're further improving ergonomics, observability, and security with Turborepo v1.2 featuring:
+
+- [**New Task Filtering API**](#new-task-filtering-api): `--filter` flag adds more expressive and intuitive task filtering capabilities to `turbo run`.
+- [**Human-readable and JSON dry runs**](#debug-and-automate-with---dry-run): `--dry-run` flag can print out information about a `turbo run` without executing any tasks, in both human and JSON-parse friendly formats.
+- [**Improved Internal Scheduler and Graph**](#improved-internal-scheduler-and-graph): We refactored `turbo` 's internal scheduler and graph to be more ergonomic and predictable
+- [**Enhanced Remote Cache Security**](#cache-outputs-integrity-and-signature-verification): Cryptographically sign remote cache artifacts with your own key phrase
```

Comment:
```diff
- phrase
+ phrase.
```

(this is separate from resolving the other comment here about using phrase at all)